### PR TITLE
Update README and use celestia-node v0.2.0

### PR DESCRIPTION
**Pull request by jbowen93**
New docker image [ghcr.io/celestiaorg/celestia-node:v0.2.0](https://github.com/celestiaorg/celestia-node/pkgs/container/celestia-node/17567122?tag=v0.2.0) was created with
```
wget https://github.com/celestiaorg/celestia-node/archive/refs/tags/v0.2.0.tar.gz
tar -xf v0.2.0.tar.gz
cd celestia-node-0.2.0
# Manually replaced the docker/Dockerfile with https://github.com/celestiaorg/celestia-node/blob/114ec3da5651a2f1ac36dd2f5d436f41e5bf6003/docker/Dockerfile to enable multi platform builds
docker buildx create --use
docker buildx build --push --platform=linux/arm64,linux/amd64 -f docker/Dockerfile -t ghcr.io/celestiaorg/celestia-node:v0.2.0 .
```

Manually tested by running
```
scripts/minimal-cevmos.sh
```
and then running commands from "Deploying a Smart Contract" section of README.md

resolves: #5 
resolves: #6 
resolves : #12

### CODE REVIEW

**Comment by jbowen93** [comment]
Keep in mind that renaming the `debug-start-cluster.sh` script to `scripts/minimal-cevmos.sh` will break the build and test GH actions in jbowen93/evmos and celestiaorg/ethermint until they're updated.

**Comment by Bidon15** [comment]
for local use, calling scripts inside the `scripts` dir is causing docker errors for finding docker-compose.yml file
Do we need to call the scripts like `./scripts/docker-xyz.yml` only ? If yes, then I'll bump the PR with updating docs path comments

**Comment by jbowen93** [comment]
> for local use, calling scripts inside the `scripts` dir is causing docker errors for finding docker-compose.yml file Do we need to call the scripts like `./scripts/docker-xyz.yml` only ? If yes, then I'll bump the PR with updating docs path comments

That's how they'll need to be called yes. Happy to update them if people have suggestions on how to handle relative paths in bash scripts better.

File: docker/light-docker-compose.yml [hunk 1]
```diff
@@ -3,14 +3,14 @@ version: '3'
 services:
   light0:
     container_name: light0
-    image: "ghcr.io/jbowen93/celestia-node:v0.1.2"
+    image: "ghcr.io/celestiaorg/celestia-node:v0.2.0"
```
Comment (Bidon15) [inline docker/light-docker-compose.yml:6]: this won't work due to a bug https://github.com/celestiaorg/celestia-node/issues/510

I propose we change the version for light as this :
```suggestion
    image: "ghcr.io/celestiaorg/celestia-node::sha-c5ee765@sha256:0c53fa3c8051a46a51d84a295edaa2621caa119c63feca48313595d6b9a765f0
```

File: docker/light-docker-compose.yml [hunk 2]
```diff
@@ -43,14 +43,14 @@ services:
         
   light2:
     container_name: light2
-    image: "ghcr.io/jbowen93/celestia-node:v0.1.2"
+    image: "ghcr.io/celestiaorg/celestia-node:v0.2.0"
```
Comment (Bidon15) [inline docker/light-docker-compose.yml:46]:
```suggestion
    image: "ghcr.io/celestiaorg/celestia-node:sha-c5ee765@sha256:0c53fa3c8051a46a51d84a295edaa2621caa119c63feca48313595d6b9a765f0"
```

File: docker/light-docker-compose.yml [hunk 3]
```diff
@@ -63,14 +63,14 @@ services:
 
   light3:
     container_name: light3
-    image: "ghcr.io/jbowen93/celestia-node:v0.1.2"
+    image: "ghcr.io/celestiaorg/celestia-node:v0.2.0"
```
Comment (Bidon15) [inline docker/light-docker-compose.yml:66]:
```suggestion
    image: "ghcr.io/celestiaorg/celestia-node:sha-c5ee765@sha256:0c53fa3c8051a46a51d84a295edaa2621caa119c63feca48313595d6b9a765f0"
```

**Comment by Bidon15** [comment]
I think we can merge after some small changes and I can work on this (later or in parallel) 
https://github.com/celestiaorg/ephemeral-cluster/issues/11

**Comment by jbowen93** [comment]
The v0.2.0 celestia-node image does actually work. 

Checking out and running this PR with `scripts/minimal-cevmos.sh` and then `docker logs light0` gives the following logs
```
2022-03-28T21:34:36.328Z	INFO	node	node/init.go:30	Initializing Light Node Store over '/celestia-light'
2022-03-28T21:34:36.328Z	INFO	node	node/init.go:66	Saving config	{"path": "/celestia-light/config.toml"}
2022-03-28T21:34:36.328Z	INFO	node	node/init.go:84	Node Store initialized
2022-03-28T21:34:36.360Z	INFO	badger	v2@v2.2007.3/levels.go:183	All 0 tables opened in 0s

2022-03-28T21:34:36.374Z	INFO	header	header/store.go:102	initialized head	{"height": 1, "hash": "54A8B66D2BEF13850D67C8D474E196BD7485FE5A79989E31B17169371B0A9C96"}
2022-03-28T21:34:36.374Z	INFO	watchdog	go-watchdog@v1.2.0/watermarks.go:16	initialized watermark watchdog policy; watermarks: []; thresholds: [67467048960 80960458752 94453868544 114693983232 121440688128 124814040576 128187393024]
2022-03-28T21:34:36.374Z	INFO	watchdog	go-watchdog@v1.2.0/watchdog.go:471	initialized heap profile capture; threshold: 0.900000; max captures: 10; dir: /celestia-light
2022-03-28T21:34:36.375Z	INFO	header	header/p2p_server.go:37	p2p-server: listening for inbound header requests
2022-03-28T21:34:36.375Z	INFO	node	node/node.go:106	started Light Node
The p2p host is listening on:
*  /ip4/192.167.10.10/tcp/2121/p2p/12D3KooWNXnkuFf51d4voY21oASCnPiobFYoQVeqrE34Kd3Wvq78
2022-03-28T21:34:36.375Z	INFO	header	header/store.go:149	loaded head	{"height": 1, "hash": "54A8B66D2BEF13850D67C8D474E196BD7485FE5A79989E31B17169371B0A9C96"}
2022-03-28T21:34:36.376Z	INFO	header	header/sync.go:251	syncing headers	{"from": 1, "to": 2}
2022-03-28T21:34:36.376Z	INFO	header	header/sync.go:268	finished syncing	{"from": 1, "to": 2, "elapsed time": 0.000000107}
2022-03-28T21:34:41.368Z	INFO	basichost	basic/natmgr.go:91	DiscoverNAT error:no NAT found
2022-03-28T21:34:41.643Z	INFO	header	header/sync.go:233	pending head	{"height": 3, "hash": "03EC2CEBD6BD0A0B1A6FBB374A28625DE1736C655B09EB3157593E92A65D4DE1"}
2022-03-28T21:34:41.644Z	INFO	header	header/sync.go:251	syncing headers	{"from": 1, "to": 3}
2022-03-28T21:34:41.645Z	INFO	header	header/store.go:273	new head	{"height": 3, "hash": "03EC2CEBD6BD0A0B1A6FBB374A28625DE1736C655B09EB3157593E92A65D4DE1"}
2022-03-28T21:34:41.645Z	INFO	header	header/sync.go:268	finished syncing	{"from": 1, "to": 3, "elapsed time": 0.000957567}
2022-03-28T21:34:41.656Z	INFO	das	das/daser.go:91	sampling successful	{"height": 3, "hash": "03EC2CEBD6BD0A0B1A6FBB374A28625DE1736C655B09EB3157593E92A65D4DE1", "square width": 2, "finished (s)": 0.011635431}
2022-03-28T21:34:56.990Z	INFO	header	header/store.go:273	new head	{"height": 4, "hash": "7CA3488033C1DC806A709CCF358553A0907B3CB068DAC9009693CC987658724F"}
2022-03-28T21:34:57.003Z	INFO	das	das/daser.go:91	sampling successful	{"height": 4, "hash": "7CA3488033C1DC806A709CCF358553A0907B3CB068DAC9009693CC987658724F", "square width": 4, "finished (s)": 0.012653087}
```

The `no chain head` error isn't present. My preference is to rely on release images, but if there's a strong preference we can use a different image. If we just want to introduce the fix for this one bug we could use [sha-36007b3](https://github.com/celestiaorg/celestia-node/pkgs/container/celestia-node/17509453?tag=sha-36007b3) instead of [sha-c5ee765](https://github.com/celestiaorg/celestia-node/pkgs/container/celestia-node/17748098?tag=sha-c5ee765) as that's the specific commit that merges the fix.

**Review by Bidon15 — APPROVED** [review]

**Comment by Bidon15** [comment]
It's actually really strange that it just works if executed as a script
But still an error can occur if execute the steps manually from the script.

**Comment by jbowen93** [comment]
> It's actually really strange that it just works if executed as a script
> 
> But still an error can occur if execute the steps manually from the script. 

Is this because of relative path issues or something else?

**Comment by Bidon15** [comment]
> > It's actually really strange that it just works if executed as a script
> > But still an error can occur if execute the steps manually from the script.
> 
> Is this because of relative path issues or something else?

i think it is more node software related rather then in docker paths/etc.

**Review by renaynay — APPROVED** [review]
utACK -- I defer to @Bidon15

File: README.md [hunk 4]
```diff
@@ -1,39 +1,50 @@
 # Ephemeral Cluster
 
-> You must have both Docker and Docker Compose (Or K8s) installed
+This repository contains contiguration files and related scripts to starting Celestia and/or Cevmos clusters using docker compose
```
Comment (renaynay) [inline README.md:3]:
```suggestion
This repository contains configuration files and related scripts to starting Celestia and/or Cevmos clusters using docker compose 
```

**Review by renaynay — COMMENTED** [review]
`scripts/full-celestia.sh` and `scripts/full-cevmos.sh` look empty.

File: README.md (hunk 4 above)
Comment (jbowen93) [reply README.md:3]: > `scripts/full-celestia.sh` and `scripts/full-cevmos.sh` look empty.

These were deleted

**Pull request merged**